packages/formation/sass/base/_b-variables.scss

```diff
@@ -83,11 +83,12 @@ $color-green-light:          #4aa564;
 $color-gold-lightest:        #fff1d2;
 $color-gold-lighter:         #fad980;
 $color-gold-light:           #f9c642;
+$color-gold-50v:             #936F38; // This is something that's not in USWDS 1.6
```

Comment:
This is copied from the [USWDS token](https://designsystem.digital.gov/design-tokens/color/system-tokens/#gold-vivid) based on an [issue discussion](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/805#issuecomment-1129955210).